### ISSUE TITLE
SSC and  THE EYE raids loot problems.

### ISSUE THREAD

**Issue by Scianana** [Atlantiss/BugTracker#9562]
I did many times **Serpentshrine Cavern** and **The eye** raids recently,  to farm the tokens for the Tier 5 set for my priest and for a shaman friend trasmog.
Unfortunately it's impossible to get the helm and the chest pieces, because Both **Lady vashj** (in SSC) and **Kael'thas** (in the eye) can't be looted.
During the combat they both suddenly go out of range for melees and when the combat is over the corpses can't be looted, the loot icon stay grey like the copse is not in the loot range.
I already asked with a ticket talking to a GM, to give me and my friend the pieces we deserve but he said "we don't give you anything at all".... just hope you can atleast fix those bosses.

thank you

[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Describe the problem

**Current behaviour**: What happens

**Expected behaviour**: What should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Revision**: Current server version (you can see current server revision by typing ".server info" command in the game chat)

**Comment by Buggingtrack** [comment]
i think whoever was tanking those bosses should have moved them, since both "teleports" to other places, even though they are actually a little more on another side, if we're talking about a precise boss position. For example kael'thas, even if he pratically floats to the air, you actually look like he is on the ground, and melees cant still him them, and the boss body when killed will appear a bit on one side instead of another, making loot impossible;
 same goes for lady vashj, she teleports to the middle of the room in the core phase, when you fight her again she just stands on place on the position before her last autoattack was made on you.
tl;dr position bosses better to ensure their bodies become unlootable.

**Comment by Piklo** [comment]
follow our [issue_template ](https://github.com/Atlantiss/BugTracker/blob/master/ISSUE_TEMPLATE.md) and read [how to use](https://github.com/Atlantiss/BugTracker#how-to-use)
